llarp/path/path.cpp

```diff
@@ -107,8 +107,33 @@ namespace llarp::path
   Path::send_path_control_message(
       std::string method, std::string body, std::function<void(oxen::quic::message m)> func)
   {
+    oxenc::bt_dict_producer btdp;
+    btdp.append("METHOD", method);
+    btdp.append("BODY", body);
+    auto payload = std::move(btdp).str();
+    auto* payload_ptr = reinterpret_cast<unsigned char*>(payload.data());
+
+    // TODO: old impl padded messages if smaller than a certain size; do we still want to?
+
+    auto crypto = CryptoManager::instance();
+
+    TunnelNonce nonce;
+    outer_nonce.Randomize();
```

Comment:
Where does this come from?

Reply:
typo, needed 2 different nonces in path build message construction and messed up copying that bit over.